src/SFA.DAS.FindEmploymentSchemes.Contentful/GdsHtmlRenderers/GdsBlockQuoteRenderer.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Contentful.Core.Models;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.FindEmploymentSchemes.Contentful.GdsHtmlRenderers
+{
+    /// <summary>
+    /// A renderer for a block quote
+    /// </summary>
+    public class GdsBlockQuoteRenderer : IContentRenderer
+    {
+        private readonly ContentRendererCollection _rendererCollection;
+
+        /// <summary>
+        /// Initializes a new GdsBlockQuoteRenderer
+        /// </summary>
+        /// <param name="rendererCollection">The collection of renderer to use for sub-content.</param>
+        public GdsBlockQuoteRenderer(ContentRendererCollection rendererCollection)
+        {
+            _rendererCollection = rendererCollection;
+        }
+
+        /// <summary>
+        /// The order of this renderer in the collection.
+        /// </summary>
+        public int Order { get; set; } = 50;
+
+        /// <summary>
+        /// Whether or not this renderer supports the provided content.
+        /// </summary>
+        /// <param name="content">The content to evaluate.</param>
+        /// <returns>Returns true if the content is a quote, otherwise false.</returns>
+        public bool SupportsContent(IContent content)
+        {
+            return content is Quote;
+        }
+
+        /// <summary>
+        /// Renders the content to a string.
+        /// </summary>
+        /// <param name="content">The content to render.</param>
+        /// <returns>The block quote as a quote HTML string.</returns>
+        public string Render(IContent content)
+        {
+            var quote = content as Quote;
+
+            var sb = new StringBuilder();
+
+            sb.Append("<div class=\"govuk-inset-text quote-and-attribution\">");
```

Comment:
Instead of making all block quote content italic, we should add the italic button to the rich text editor and mark italic text that way instead.